preprocess/analysis/mapping.py

```diff
@@ -0,0 +1,49 @@
+"""
+@Author: Scarlet Nguyen 
+"""
+
+
+import csv
+
+from language_analysis import analyze_entities, analyze_sentiment
+
+def get_analysis(text):
+  return {
+    'sentiment': analyze_sentiment(text),
+    'entities': analyze_entities(text)
+  }
+
+file = open('input.csv')
```

Comment:
Don't just dump your Python code as raw lines to be execute in the file, use if __name__ == '__main__' unless there's some reason not to